sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -36,7 +38,7 @@ export class Application extends RecordDataModel {
     type: "jsonb",
     nullable: false,
   })
-  data: any;
+  data: ApplicationData;
```

Comment:
my story also has this point. But it's partial right? shouldn't it be `Partial<ApplicationData>`

Reply:
mm, I am not clear, may be we can syncup

Reply:
As we talked, partial does not apply to this context. Please let me know if you find something that indicates otherwise.